src/pysdmx/model/dataset.py

```diff
@@ -0,0 +1,22 @@
+"""Dataset module."""
+
+from typing import Any, Dict
+
+from msgspec import Struct
+import pandas as pd
+
+
+class Dataset(Struct, frozen=False, kw_only=True):
```

Comment:
I have several comments about this Dataset class but let's discuss them in the afternoon, as agreed :).

Reply:
Agreed on meeting #72